man/lcovrc.5

```diff
@@ -884,14 +884,23 @@ Default is 1.
 .PP
 
 .BR lcov_branch_coverage " ="
-.IR 0 | 1
+.IR 0 | 1 | exclude-exceptions
 .IP
```

Comment:
As mentioned above, please use a separate configuration file directive for specifying whether exception branches should be included or not.

Reply:
Done